digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/components/NotificationApp.vue

```diff
@@ -24,102 +24,128 @@
+          </v-row>
         </v-navigation-drawer>
       </v-layout>
     </v-container>
   </v-app>
 </template>
 <script>
+  import * as  notificationlAPI from '../notificationsAPI';
   export default {
     data () {
       return {
         drawerNotification: null,
-        items: [
-          {
-            avatar: 'https://cdn.vuetifyjs.com/images/lists/3.jpg',
-            title: "<span class='primary-color--text' span style='font-weight: bold'>Sara</span>  has commented on your task in <span style='font-weight: bold'>Maintenance Board</span> project",
-            subtitle: "3 hours ago",
-            background:"aliceblue",
-          },
-          {
-            avatar: 'https://cdn.vuetifyjs.com/images/lists/2.jpg',
-            title: "<span class='primary-color--text' span style='font-weight: bold'>Hamadi</span>  has commented on your task in <span style='font-weight: bold'>Maintenance Board</span> project",
-            subtitle: "1 day ago",
-            background:"",
-
-          },
-        ],
+        items: {},
         mini: false,
-        right: null
+        right: null,
+        badge: 0,
+        notificationAction: {
+          id: "",
+          operation: ""
+        }
+      }
+    },
+    created() {
+      this.getNotifListItem();
+      /*window.require(['SHARED/DWNotificationPopover'], function(dwNotificationPopover) {
+        dwNotificationPopover.init();*/
+      });
+    },
+    methods: {
+      getNotifListItem() {
+        notificationlAPI.getNotificationItems().then(
+                (data) => {
+                  this.items = data.notifications;
+                  this.badge = data.badge;
+                }
+        )
+      },
+      getActivityId(id) {
+            const dataLink = $(`#${id}`).find('.contentSmall:first').data('link');
+            const linkId = dataLink.split('/portal/intranet/');
+            const dataId = $(`#${id}`).find('li').data('id');
+            this.notificationAction.id = dataId.toString();
+            this.notificationAction.operation = 'markAsRead';
+            notificationlAPI.updateNotifications(this.notificationAction);
+            if(linkId.length >1 ) {
+              location.href = `${eXo.env.portal.context}/${eXo.env.portal.portalName}/${linkId[1]}`;
+            } else {
+              const id = linkId[0].split('/view_full_activity/')[1];
+              location.href = `${eXo.env.portal.context}/${eXo.env.portal.portalName}/activity?id=${id}`;
+            }
+      },
+      notificationOperation(id, operation) {
+        this.notificationAction.id = id;
+        this.notificationAction.operation = operation;
+        notificationlAPI.updateNotifications(this.notificationAction);
+      },
+      hideNotification(id) {
+        const dataId = $(`#${id}`).find('li').data('id');
+        this.notificationAction.id = dataId.toString();
+        this.notificationAction.operation = 'hide';
+        notificationlAPI.updateNotifications(this.notificationAction);
```

Comment:
```suggestion
        notificationlAPI.updateNotification(dataId.toString(), 'hide');
```
- Verify if .toString() is needed

Reply:
ok